int16 mode
float64 ttime
float64 teta1
float64 teta2
float64 teta3
---
int16 status
